### PR TITLE
Create contributor-interview-amirhossein.md

### PR DESCRIPTION
**Pull request by l-delin**

### CODE REVIEW

File: content/news/posts/2023/contributor-interview-amirhossein.md
Comment (Julievrz) [inline content/news/posts/2023/contributor-interview-amirhossein.md]:
```suggestion
<img style="border: 1px solid #CCC; float: center; margin: 0 1em 1em 0;" width="517" height="388" src="static/assets/images/2023/03/amirhossein.png">
```

File: content/news/posts/2023/contributor-interview-amirhossein.md
Comment (Julievrz) [inline content/news/posts/2023/contributor-interview-amirhossein.md]:
```suggestion
image: static/assets/images/2023/03/amirhossein.png
```

**Comment by l-delin** [comment]
Contributor interviews are back 🎉🐧  Thanks @Julievrz 

Ping @coffrep for review

**Comment by l-delin** [comment]
Hello @PrestaShop/prestashop-maintainers, could any of you check and tell me if the picture is working? When I click on it, it redirects to the wrong repo but  I know the picture is in the exact folder I specified, same branch, everything:

image

image

File: content/news/posts/2023/contributor-interview-amirhossein.md
Comment (coffrep) [inline content/news/posts/2023/contributor-interview-amirhossein.md]:
```suggestion
Fasleaval, on the other hand, is an agency specializing in PrestaShop products and services. We have provided tens of hours of [free training videos](https://fasleaval.com/blog/آموزش-پرستاشاپ-رایگان-ویدیوی) from downloading PrestaShop up to some more advanced configurations. We also offer some themes and modules.
```

File: content/news/posts/2023/contributor-interview-amirhossein.md
Comment (coffrep) [inline content/news/posts/2023/contributor-interview-amirhossein.md]:
```suggestion
I have also helped to teach PrestaShop in high schools in Iran. There’s one chapter in textbooks dedicated to PrestaShop and tens of thousands of students learn how to use PrestaShop every year since 2018.
```

File: content/news/posts/2023/contributor-interview-amirhossein.md
Comment (coffrep) [inline content/news/posts/2023/contributor-interview-amirhossein.md]:
```suggestion
And that’s how, less than a year later, Fasleaval was founded. We started uploading free PrestaShop training videos and I guess many people have used those videos so far to start their own ecommerce by PrestaShop.
```

**Review by coffrep — CHANGES_REQUESTED** [review]
Hi l-delin, I've added very minor suggestions, trying to keep the interview as original as possible in the member's own tone.

**Comment by l-delin** [comment]
Thank you @coffrep :-) I'm waiting for feedback regarding the picture in the article and once that's been fixed, I think the article will be ready to go

**Comment by l-delin** [comment]
Hello @coffrep, could we get this article published this week or the next one? :-) 
I'll submit the Crowdin Monthly later this week, it would be nice if the monthly's were published this week (it'll be posted a bit late otherwise)

**Comment by coffrep** [comment]
Hi @l-delin, great news! Yes we can publish this contributor interview either this week or next week. Please do submit it when it is good for you.

**Comment by l-delin** [comment]
Hi, that's perfect. Can it be published tomorrow, on 04/12? That way I can talk about the interview on the next Crowdin Monthly. Thank you!

**Review by coffrep — APPROVED** [review]

**Comment by l-delin** [comment]
Hello, could I please get a merge @kpodemski ? :-) Thank you!

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thanks @l-delin @coffrep :)

**Pull request merged**
